app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -55,6 +55,9 @@ public enum SenderSyncMessageStatus {
 	@Access(AccessType.FIELD)
 	private Date dateSent;
 	
+	@Column(name = "event_date")
```

Comment:
i think this should be required, and for existing rows at upgrade you will need to add a custom changeset to match the value of date_created of the same row

Reply:
it can be required, but creating a change set to update existing rows i think there no need to do that as we don't have that table in productions

Reply:
ok